.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     'no-unused-vars': 'warn',
     'react/no-unknown-property': 0,
     'react/prop-types': 0,
-    'react/react-in-jsx-scope': 0
+    'react/react-in-jsx-scope': 0,
+    'react-hooks/rules-of-hooks': 0
```

Comment:
We should be careful about this setting. I would like to discuss about this changes with all developers.